extension/first-run.html

```diff
@@ -9,6 +9,27 @@
     <script src="/js/first-run.js"></script>
   </head>
   <body class="fx-relay-first-run">
+    <div class="privacy-modal modal hide-modal">
+      <div class="modal-content">
+        <h1 class="modal-h1">Privacy</h1>
+        <div class="modal-copy">
+          <p>May we collect the following anonymous data about your use of Firefox Relay?</p>
+          <ul class="collected-data-list">
+            <li class="data-list-item">When the add-on pop-up is viewed</li>
+            <li class="data-list-item">When a button or link in the add-on is clicked</li>
+            <li class="data-list-item">When the Relay icon is injected into website forms</li>
+            <li class="data-list-item">When a new alias is created</li>
```

Comment:
Apologies, I haven't been following all the content changes in this repo, but is this "alias" or "address" these days?

Reply:
"alias" is the latest hotness.